test/CMakeLists.txt

```diff
@@ -514,6 +517,14 @@ if(iodaconv_bufr_ENABLED)
                             gdas.t06z.adpsfc.tm00.nc ${IODA_CONV_COMP_TOL_ZERO}
                     DEPENDS bufr2ioda.x )
 
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_m_x_n
```

Comment:
is there a better name? this means almost nothing to me... Or perhaps describe what the test does in the YAML?

Reply:
Fixed Spelling.